dali/operators/reader/loader/webdataset/tar_utils.h

```diff
@@ -62,15 +62,30 @@ class DLL_PUBLIC TarArchive {
    */
   bool EndOfArchive() const;
 
+  enum EntryType {
```

Comment:
Shouldn't this be `enum class`? I'm not sure here, but I believe simple `enum` makes it global.

Reply:
it doesn't